src/firestore/indexes-sort.ts

```diff
@@ -79,7 +79,11 @@ export function compareApiField(a: API.Field, b: API.Field): number {
     return aName.fieldPath.localeCompare(bName.fieldPath);
   }
 
-  return compareArrays(a.indexConfig.indexes || [], b.indexConfig.indexes || [], compareApiIndex);
```

Comment:
Previously `compareArrays` implicitly sorted all arguments first.  Now it does not, so we change two calls to do it explicitly.